molecule/resources/tests/utils.py

```diff
@@ -0,0 +1,18 @@
+# Enumerate instead of parsing so we pick up errors if a host is added/changed
+ver_lookup = {
+    "postgresql-96-c7": ("9.6", "centos"),
+    "postgresql-10-c7": ("10", "centos"),
+    "postgresql-11-c7": ("11", "centos"),
+
+    "postgresql-96-u1804": ("9.6", "ubuntu"),
```

Comment:
this lookup will need to be reviewed when we add support for centos 8 and ubuntu 20.04

Reply:
It might not be necessary if the postgres package has the same layout on disk across distro versions

Reply:
let see when the time comes